src/daemon/revaultd.rs

```diff
@@ -344,7 +342,11 @@ impl RevaultD {
             config.unvault_csv,
         )?;
         let cpfp_descriptor = cpfp_descriptor(managers_pubkeys)?;
-        let emergency_address = EmergencyAddress::from(config.emergency_address)?;
+        let emergency_address = if let Some(conf) = config.stakeholder_config {
+            Some(EmergencyAddress::from(conf.emergency_address)?)
```

Comment:
I think it is better to throw an error during the deserialization of the config than in the daemon. see https://github.com/re-vault/revaultd/pull/84/files#r576995072

Reply:
I agree, i was being lazy :) Will amend.